schedule/create_hdd_autoyast_pc.yaml

```diff
@@ -0,0 +1,12 @@
+---
+name: create_hdd_autoyast_wicked
+vars:
+  AUTOYAST: autoyast_sle15/pc_tools.xml
+  DESKTOP: textmode
+  HDDSIZEGB: 30
```

Comment:
just curious, why 30GB?

Reply:
historical reasons :D but I am open for other values ... in case you will explain them :D

Reply:
to give proper explanations we should check how much disk is needed once everything is installed, so I don't have a good estimation now. We can change it later if we see the need..